src/misc/cyfs-monitor/src/case/cyfs_repo.rs

```diff
@@ -55,7 +55,7 @@ impl MonitorRunner for CyfsRepoMonitor {
                 config.conn_strategy = cyfs_client::ConnStrategy::TcpOnly;
                 let mut client = NamedCacheClient::new(config);
                 client.init().await?;
-                self.cyfs_client.set(client);
+                let _ = self.cyfs_client.set(client);

```

Comment:
if let Err(_) = self.cyfs_client.set(client) {
    unreachable!("......");
}

Reply:
I don't known what it is, Maybe I should leave the warning currently.